### PR TITLE
New kits

### PR DESCRIPTION
**Pull request by katnep**
so this part here makes it so that fulton packs can be used on people instead of just objects(however you cannot remove yourself), removes the medevac packs because they are now redundant, and adds a bundle of survival pens for 1000 (which at this point i think might be too low so i'm gonna boost that up to at least 1500)

### CODE REVIEW

**Comment by katnep** [comment]
ok somehow this pushed my fulton changes too
no idea how that happened

File: code/modules/mining/machine_vending.dm [hunk 1]
```diff
@@ -24,14 +24,14 @@
...
+		new /datum/data/mining_equipment("Survival Medipen",	/obj/item/weapon/reagent_containers/hypospray/medipen/survival,			400),
 		new /datum/data/mining_equipment("Brute First-Aid Kit",	/obj/item/weapon/storage/firstaid/brute,						   		600),
 		new /datum/data/mining_equipment("Tracking Implant Kit",/obj/item/weapon/storage/box/minertracker,                              600),
 		new /datum/data/mining_equipment("Jaunter",             /obj/item/device/wormhole_jaunter,										750),
 		new /datum/data/mining_equipment("Kinetic Crusher", 	/obj/item/weapon/twohanded/required/mining_hammer,						750),
 		new /datum/data/mining_equipment("Kinetic Accelerator", /obj/item/weapon/gun/energy/kinetic_accelerator,               	   		750),
 		new /datum/data/mining_equipment("Resonator",           /obj/item/weapon/resonator,                                    	   		800),
-		new /datum/data/mining_equipment("Medivac Balloon",     /obj/item/weapon/extraction_pack/medivac,                               800),
+		new /datum/data/mining_equipment("Survival bundle",		/obj/item/weapon/storage/box/medipens/utility1,	 				  		800),
```
Comment (ghost) [inline code/modules/mining/machine_vending.dm]: 1000 - 1200 points seems a bit more fair

**Review by ghost — APPROVED** [review]

**Comment by TalkingCactus** [comment]
So since this changes the same files as your other PR, when one gets merged you'll end up with conflicts. It's best to have all of these changes under one branch and PR.

**Comment by katnep** [comment]
yeah i only realised it changes the same files after i made the PR

File: code/modules/mining/machine_vending.dm [hunk 2]
```diff
@@ -24,14 +25,15 @@
 		new /datum/data/mining_equipment("Motion Tracker",		/obj/item/device/t_scanner/motionTracker	 ,                     		400),
 		new /datum/data/mining_equipment("GAR scanners",		/obj/item/clothing/glasses/meson/gar,					  		   		500),
 		new /datum/data/mining_equipment("Explorer's Webbing",	/obj/item/weapon/storage/belt/mining,									500),
-		new /datum/data/mining_equipment("Survival Medipen",	/obj/item/weapon/reagent_containers/hypospray/medipen/survival,			500),
+		new /datum/data/mining_equipment("Survival Medipen",	/obj/item/weapon/reagent_containers/hypospray/medipen/survival,			400),
```
Comment (TalkingCactus) [inline code/modules/mining/machine_vending.dm]: Why was this price reduced? These are way too powerful to begin with.

File: code/modules/mining/machine_vending.dm [hunk 3]
```diff
@@ -24,14 +25,15 @@
...
 		new /datum/data/mining_equipment("Brute First-Aid Kit",	/obj/item/weapon/storage/firstaid/brute,						   		600),
 		new /datum/data/mining_equipment("Tracking Implant Kit",/obj/item/weapon/storage/box/minertracker,                              600),
 		new /datum/data/mining_equipment("Jaunter",             /obj/item/device/wormhole_jaunter,										750),
 		new /datum/data/mining_equipment("Kinetic Crusher", 	/obj/item/weapon/twohanded/required/mining_hammer,						750),
 		new /datum/data/mining_equipment("Kinetic Accelerator", /obj/item/weapon/gun/energy/kinetic_accelerator,               	   		750),
 		new /datum/data/mining_equipment("Resonator",           /obj/item/weapon/resonator,                                    	   		800),
-		new /datum/data/mining_equipment("Medivac Balloon",     /obj/item/weapon/extraction_pack/medivac,                               800),
+		new /datum/data/mining_equipment("whetstone",           /obj/item/weapon/sharpener,                                             1000),
+		new /datum/data/mining_equipment("Survival bundle",		/obj/item/weapon/storage/box/medipens/utility1,	 				  		1000),
```
Comment (TalkingCactus) [inline code/modules/mining/machine_vending.dm]: If this is a box of survival pens, you may want to increase the price and capitalize+clarify the title. Maybe 
`Survival Medipen Bundle`

**Pull request merged**